2.evaluate_data/compute_sc_anomaly_data/compute_all_sc_anomaly_data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This script calls python code to compute single-cell anomaly data with all datasets.
+
+conda init bash
+conda activate jump_sc
+
+# Get the root directory
+git_root=$(git rev-parse --show-toplevel)
+
+py_path="nbconverted"
+
+jupyter nbconvert --to python --output-dir="${py_path}/" *.ipynb
```

Comment:
Consider using `shellcheck` on this, which is a part of the pre-commit suite of tooling in this repo. A suggestion from that linter: consider using a relative path here to more safely specify the glob input (rule [SC2035](https://www.shellcheck.net/wiki/SC2035)).

```suggestion
jupyter nbconvert --to python --output-dir="${py_path}/" ./*.ipynb
```

Reply:
I can see how this might be unsafe in some programs, so I'll change it